doc/api_data.json

```diff
@@ -0,0 +1 @@
  },
  {
    "type": "get",
    "url": "/users/:id",
    "title": "Retrieve User",
    "name": "GetUser",
    "group": "User",
    "parameter": {
      "fields": {
        "Parameter": [
          {
            "group": "Parameter",
            "type": "Number",
            "optional": false,
            "field": "id",
            "description": "<p>Users unique ID.</p>"
          }
        ]
      }
    },
    "success": {
      "fields": {
        "Success 200": [
          {
            "group": "Success 200",
            "type": "String",
            "optional": false,
            "field": "_id",
            "description": "<p>Id of the User.</p>"
          },
          {
            "group": "Success 200",
            "type": "String",
            "optional": false,
            "field": "familyName",
            "description": "<p>Last Name of the User.</p>"
          },
          {
            "group": "Success 200",
            "type": "String",
            "optional": false,
            "field": "imageUrl",
            "description": "<p>URL to the Users Gravatar.</p>"
          },
          {
            "group": "Success 200",
            "type": "String",
            "optional": false,
            "field": "name",
            "description": "<p>Name of the User.</p>"
          },
          {
            "group": "Success 200",
            "type": "String",
            "optional": false,
            "field": "googleId",
            "description": "<p>GoogleId of the User.</p>"
          },
          {
            "group": "Success 200",
            "type": "String",
            "optional": false,
            "field": "email",
            "description": "<p>Email of the User.</p>"
          },
          {
            "group": "Success 200",
            "type": "String",
            "optional": false,
            "field": "givenName",
            "description": "<p>First name of the User.</p>"
          }
        ]
      },
      "examples": [
        {
          "title": "Success-Response:",
          "content": "HTTP/1.1 200 OK\n{\n  \"_id\": \"5c8834497a09b7bf2ca1d98b\",\n  \"familyName\": \"Chia\",\n  \"imageUrl\": \"https://lh3.googleusercontent.com/-CIJl6ntqVg4/AAAAAAAAAAI/AAAAAAAAABY/Jfd1E76bbhA/s96-c/photo.jpg\",\n  \"name\": \"Gideon Chia\",\n  \"googleId\": \"116550101409368732312\",\n  \"email\": \"gideon.chia@gmail.com\",\n  \"givenName\": \"Gideon\"\n}",
          "type": "json"
        }
      ]
    },
    "version": "0.0.0",
    "filename": "api/user_api.js",
    "groupTitle": "User"
  },
  {
    "type": "get",
    "url": "/users",
    "title": "Retrieve All Users",
    "name": "GetUsers",
    "group": "User",
    "success": {
      "examples": [
        {
          "title": "Success-Response:",
          "content": "HTTP/1.1 200 OK\n[\n  {user-object},\n  {user-object}\n]",
          "type": "json"
        }
      ]
    },
    "version": "0.0.0",
    "filename": "api/user_api.js",
    "groupTitle": "User"
  }
]
```

Comment:
Can we not put personal information such as the google id of the user and email and name in this file?